src/SageFLBuilder.php

```diff
@@ -38,15 +36,12 @@ final class SageFLBuilder
         Breadcrumbs::class,
         Button::class,
         ContentImage::class,
-        EventsArchive::class,
         FilterBar::class,
         Gallery::class,
-        God::class,
         GravityForm::class,
         HomeThemeLayout::class,
         PageHeading::class,
         PageSlider::class,
-        PostGrid::class,
         RichText::class,
```

Comment:
This class has been deleted though?

Reply:
Yes :/
![Screenshot_16](https://user-images.githubusercontent.com/6186169/83729369-282f3b80-a650-11ea-835e-8f212dd08c8f.png)

Reply:
You deleted it here: https://github.com/ItinerisLtd/sage-flbuilder/pull/112/files#diff-82b9885e426bbded65b8f55a1f129a73
But you are still referencing it

Reply:
@codepuncher , yes. I'm blind. fixed